src/tests/JIT/HardwareIntrinsics/Arm/Shared/SveSimpleNoOpTest.template

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Why is this not named as `SveCreateFalseMaskTest.template`?

Reply:
Are you planning to reuse it for other APIs?

Reply:
This was my suggestion - I'd hope we can use this one for future APIs. Whereas the `SveCreateTrueMaskTest.template` is specific to true mask due to the handling.